great_expectations/rule_based_profiler/parameter_builder/metric_single_batch_parameter_builder.py

```diff
@@ -101,7 +108,7 @@ def _build_parameters(
             {
                 FULLY_QUALIFIED_PARAMETER_NAME_VALUE_KEY: None
                 if parameter_node[FULLY_QUALIFIED_PARAMETER_NAME_VALUE_KEY] is None
-                else parameter_node[FULLY_QUALIFIED_PARAMETER_NAME_VALUE_KEY][0],
+                else parameter_node[FULLY_QUALIFIED_PARAMETER_NAME_VALUE_KEY][-1],
```

Comment:
Why are we changing this from the first one to the last one? Could you add a comment with the reason?

Reply:
@billdirks We stylistically want the most recent batch -- this does not matter, but "stronger" style.